ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -51,54 +52,54 @@ describe('AdminInstitutionEditSpec', () => {
       .toContain('Display name must be 80 characters or less');
   });
 
-  it('should hide/show controlled card when controlled tier disabled/enabled', async() => {
+  it('should hide/show CT card details when controlled tier disabled/enabled', async() => {
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
     expect(wrapper).toBeTruthy();
 
-    const ctEnabledToggle = wrapper.find('[data-test-id="ct-enabled-switch"]').first().instance() as InputSwitch;
+    const ctEnabledToggle = wrapper.find('[data-test-id="controlled-enabled-switch"]').first().instance() as InputSwitch;
```

Comment:
changes in this file are almost entirely like this.  Regularizing the naming of these helped me make the components more generic.